1.33.2

matteobruni opened the pull request:
# tsParticles 1.33.2

## Bug Fixes

- Fixed svelte component, Issue #1924 
- Improved Absorbers
- Improved Orbits

matteobruni committed:
chore(orbit): improved orbit animation and output
build: updated orbit demo

matteoums committed:
build: updating absorbers

matteobruni committed:
fix: brought v2 changes to absorbers

matteobruni committed:
fix: fixed svelte component, reverted to an older svelte version for now, closes #1924

ghost commented:
## DeepCode's analysis on [#744d1e](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/16f9aa9d23d68b53ed555924e6ab4e0be8f25f22/matteobruni/tsparticles/744d1ec9bd59a797bda7c4d5ea87293893994acd/pr/_/%2F/code/?utm_source=gh_review&c=0&w=2&i=0&) found:

- :warning: **2** warnings :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">Testing a collection size for < 0 will always be false. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/744d1ec9bd59a797bda7c4d5ea87293893994acd/engine/src/Plugins/Absorbers/AbsorberInstance.ts#L98">AbsorberInstance.ts:98</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/16f9aa9d23d68b53ed555924e6ab4e0be8f25f22/matteobruni/tsparticles/744d1ec9bd59a797bda7c4d5ea87293893994acd/pr/_/%2Fengine%2Fsrc%2FPlugins%2FAbsorbers%2FAbsorberInstance.ts/javascript%2Fdc_interfile_project%2FCollectionArraySizeMatch/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">Two subsequent branches of an if statement have duplicate bodies (here and here). This may be caused by a copy-paste error. If this usage is intentional, consider merging the branches to avoid code duplication. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/744d1ec9bd59a797bda7c4d5ea87293893994acd/engine/src/Plugins/Absorbers/plugin.ts#L44">plugin.ts:44</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/16f9aa9d23d68b53ed555924e6ab4e0be8f25f22/matteobruni/tsparticles/744d1ec9bd59a797bda7c4d5ea87293893994acd/pr/_/%2Fengine%2Fsrc%2FPlugins%2FAbsorbers%2Fplugin.ts/javascript%2Fdc_interfile_project%2FDuplicateIfBody/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/16f9aa9d23d68b53ed555924e6ab4e0be8f25f22/matteobruni/tsparticles/744d1ec9bd59a797bda7c4d5ea87293893994acd/pr/_/%2F/code/?utm_source=gh_review&c=0&w=2&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)
#### 👉 The DeepCode service and API will be deprecated in August, 2021. [Here](https://www.deepcode.ai/blog/migrate-to-snyk-code) is the information how to migrate. Thank you for using DeepCode 🙏 ❤️ !
If you are using our plugins, you might be interested in their successors: [Snyk's JetBrains plugin](https://snyk.co/udpkq) and [Snyk's VS Code plugin](https://snyk.co/udpkr).

matteobruni committed:
chore(release): published new version

 - ng-particles@2.16.2
 - inferno-particles@1.16.2
 - jquery-particles@1.33.2
 - preact-particles@1.33.2
 - react-tsparticles@1.33.2
 - riot-particles@1.7.2
 - solid-particles@1.16.2
 - svelte-particles@1.16.2
 - particles.vue@2.16.2
 - particles.vue3@1.16.2
 - web-particles@1.6.2
 - ng-particles-demo@1.18.2
 - editor-demo@1.18.2
 - imports-ts-latest@1.4.2
 - imports-ts34@1.4.2
 - imports-ts38@1.4.2
 - imports-ts40@1.4.2
 - inferno-particles-demo@1.16.2
 - ionic@0.2.2
 - jquery-particles-demo@1.18.2
 - preact-particles-demo@1.18.2
 - react-tsparticles-demo@1.18.2
 - riot-particles-demo@1.7.2
 - solid-particles-demo@1.16.2
 - svelte-demo@1.16.2
 - tsparticles-demo@1.18.2
 - tsparticles-demo-new@1.18.2
 - particles.vue-demo@1.18.2
 - particles.vue3-demo@1.16.2
 - web-particles-demo@1.18.2
 - tsparticles-editor@1.33.2
 - tsparticles@1.33.2
 - tsparticles-interaction-light@1.33.2
 - tsparticles-interaction-particles-repulse@1.33.2
 - tsparticles-plugin-infection@1.33.2
 - tsparticles-preset-big-circles@1.18.2
 - tsparticles-preset-bubbles@1.18.2
 - tsparticles-preset-confetti@1.18.2
 - tsparticles-preset-fire@1.18.2
 - tsparticles-preset-firefly@1.18.2
 - tsparticles-preset-fireworks@1.18.2
 - tsparticles-preset-fountain@1.18.2
 - tsparticles-preset-links@1.18.2
 - tsparticles-preset-sea-anemone@1.18.2
 - tsparticles-preset-snow@1.18.2
 - tsparticles-preset-stars@1.18.2
 - tsparticles-shape-bubble@1.18.2
 - tsparticles-shape-heart@1.18.2
 - tsparticles-shape-multiline-text@1.18.2
 - tsparticles-shape-rounded-rect@1.18.2
 - tsparticles-shape-spiral@1.18.2
 - cra-template-particles-typescript@1.16.2
 - cra-template-particles@1.16.2
 - tsparticles-updater-orbit@1.33.2
 - tsparticles-website@1.33.2

matteobruni committed:
Merge branch 'main' into dev
